app/controllers/concerns/first_item_from_next_page.rb

```diff
@@ -0,0 +1,12 @@
+module FirstItemFromNextPage
+  extend ActiveSupport::Concern
+
+private
+
+  def first_item_from_next_page(collection, per_page)
+    has_next_page_available = collection.size > per_page
+    if has_next_page_available
+      collection[per_page]
+    end
+  end
+end
```

Comment:
Would it be worth merging `FirstItemFromNextPage` and `NextLinkHeader`? Maybe call it `ChangedSincePagination`? One-method per concern just feels a bit fine-grained to me, unless you're following some best-practice advice I haven't seen?